tests/reftests/abspos/abspos-containing-block-initial-004-ref.html

```diff
@@ -0,0 +1,3 @@
+<!doctype html>
+<html>
+<div style="position:absolute;top:100px;background-color:blue;left:100px;width:100px;height:100px">
```

Comment:
Is this the same as the test but with properties in e different order? I’d prefer if the reference did not use abspos at all. (But e.g. margins, instead)